enable/gcbench/bench.py

```diff
@@ -0,0 +1,129 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+import importlib
+import inspect
+import os
+import time
+
+import numpy as np
+
+_MAX_DURATION = 1.0
+_SIZE = (512, 512)
```

Comment:
Should we be working with larger screen sizes e.g. (1024, 1024) given that we are regularly working with larger screens now? Not sure how much of an effect that will have on the benchmarking.

Reply:
Yeah... Does this might lead to better results since you're drawing more pixels and spending more time in lower-level non-Kiva code. Only downside I see is that it results in larger images which we plan to upload along with the HTML output (when adding benchmarking results to documentation). We maybe don't need to worry about it too much. Images can be made smaller by making them easier to compress ("less noise").